src/styles/utils/positionStyle.ts

```diff
@@ -23,6 +23,7 @@ export type PositionStyleProps = {
   $right?: ResponsiveValues<PositionSpacing>;
   $bottom?: ResponsiveValues<PositionSpacing>;
   $left?: ResponsiveValues<PositionSpacing>;
+  $inset?: ResponsiveValues<PositionSpacing>;
```

Comment:
No positioning toolkit is complete without `inset`

Reply:
i think my prs need this level of description